src/utils/index.js

```diff
@@ -91,22 +91,20 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+// eslint-disable-next-line consistent-return
```

Comment:
this method should return the server response or throw an error, I don't see reasons for inconsistency